src/main/java/com/matt/forgehax/mods/DerpMod.java

```diff
@@ -0,0 +1,244 @@
+package com.matt.forgehax.mods;
+
+import com.matt.forgehax.asm.events.PacketEvent;
+import com.matt.forgehax.events.LocalPlayerUpdateEvent;
+import com.matt.forgehax.util.command.Setting;
+import com.matt.forgehax.util.mod.Category;
+import com.matt.forgehax.util.mod.ToggleMod;
+import com.matt.forgehax.util.mod.loader.RegisterMod;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.network.Packet;
+import net.minecraft.network.play.client.CPacketAnimation;
+import net.minecraft.network.play.client.CPacketEntityAction;
+import net.minecraft.network.play.client.CPacketPlayer;
+import net.minecraft.util.EnumHand;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import java.util.Random;
+
+import static com.matt.forgehax.Helper.getLocalPlayer;
+import static com.matt.forgehax.Helper.getModManager;
+import static com.matt.forgehax.util.PacketHelper.ignoreAndSend;
+import static com.matt.forgehax.util.PacketHelper.isIgnored;
+
+/**
+ * Coded by LoganDark on 8/21/2019
+ */
+@RegisterMod
+public class DerpMod extends ToggleMod {
+  public DerpMod() {
+    super(Category.PLAYER, "Derp", false, "Derp");
+  }
+
+  private float error;
+  private boolean sneaking;
+  private TimerMod timerMod;
+
+  @Override
+  protected void onEnabled() {
```

Comment:
Move this method down to be above `::onDisabled`

Reply:
Will be in the next commit